### PR TITLE
Adding the guest conf.d writing

### PR DESCRIPTION
**Pull request by hub-cap**
- Made sure to append the conf in the startup
- Added the methods to append/write
- Made the dbaas guest agent write guest uuid

### CODE REVIEW

File: bin/reddwarf-guestagent [hunk 1]
```diff
@@ -53,6 +53,10 @@ if __name__ == '__main__':
     try:
         conf, app = config.Config.load_paste_app('reddwarf-guestagent',
                                                  options, args)
+        # Use the config file location for putting the new config values
+        conf_loc = '%s/%s' % (config.Config.get('here'), 'conf.d/guest_info')
```
Comment (cp16net) [inline bin/reddwarf-guestagent:57]: what is here? am i here? who's there?

**Comment by cp16net** [comment]
+1

File: etc/reddwarf/conf.d/README [hunk 2]
```diff
@@ -0,0 +1,4 @@
+These conf files, are read and used by the guest to provide extra
```
Comment (ed-) [inline etc/reddwarf/conf.d/README]: nix that comma

File: reddwarf/common/config.py [hunk 3]
```diff
@@ -43,5 +45,34 @@ def load_paste_config(cls, *args, **kwargs):
...
     @classmethod
+    def append_to_config_values(cls, *args):
+        config_file = openstack_config.find_config_file(*args)
+        if not config_file:
+            raise RuntimeError("Unable to locate any configuration file. "
+                                "Cannot load application %s" % app_name)
+        # Now take the conf file values and append them to the current conf
+        f = open(config_file, 'r')
+        for line in f:
+                m = re.match("\s*([^#]\S+)\s*=\s*(\S+)\s*", line)
```
Comment (ed-) [inline reddwarf/common/config.py]: I wonder if split/strip wouldn't really be easier to understand than what appears to be an incantation to Yig, the Father of Serpents there

File: etc/reddwarf/conf.d/README (hunk 2 above)
Comment (hub-cap) [inline etc/reddwarf/conf.d/README]: done.

**Comment by ed-** [comment]
+1 leave the incantation to Yig as it is

File: bin/reddwarf-guestagent (hunk 1 above)
Comment (TimSimpsonR) [inline bin/reddwarf-guestagent:57]: Here is what you are, and who are there.

File: reddwarf/common/config.py [hunk 4]
```diff
@@ -43,5 +45,34 @@ def load_paste_config(cls, *args, **kwargs):
         return conf
 
     @classmethod
+    def append_to_config_values(cls, *args):
+        config_file = openstack_config.find_config_file(*args)
+        if not config_file:
+            raise RuntimeError("Unable to locate any configuration file. "
+                                "Cannot load application %s" % app_name)
+        # Now take the conf file values and append them to the current conf
+        f = open(config_file, 'r')
```
Comment (TimSimpsonR) [inline reddwarf/common/config.py]: Even with something this simple lets avoid one letter variable names. At the very least it's a pylint violation (I think pylint even complains with for loops).

Comment (ed-) [inline reddwarf/common/config.py]: I think you might be more comfortable with the `with open(file, 'r') as inf` pattern here and elsewhere.

File: bin/reddwarf-guestagent (hunk 1 above)
Comment (ed-) [inline bin/reddwarf-guestagent:57]:
```
What the hell am I looking at?! When does this happen in the movie?!
"Now". You're looking at "now", sir. Everything that happens now is happening "now".
What happened to "then"?
We passed "then".
When!?
Just now. Were at "now," now.
Go back to "then"!
When?
Now!
"Now?"
Now!
I can't.
Why!?
We missed it.
When!?
Just now.
... When will "then" be "now"?
Soon.
```

**Pull request merged**